src/main/java/Duke.java

```diff
@@ -1,10 +1,90 @@
+
+import java.util.Scanner;
 public class Duke {
+    public static void printLines(){
+        System.out.println("--------------------------------------------------");
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?\n");
+
+        Scanner ReadInput = new Scanner(System.in);
```

Comment:
"ReadInput" is a variable, and should be in camelCase as per the naming convention